Develop3

dashenya opened the pull request.

Commit:
added pages and page components

Commit:
added HelpingMethods

Commit:
added rules for webdriver

Commit:
added tests

Commit:
added pom.xml

Commit:
add allure report

mkouznetsov-git commented on src/test/java/ConstructorTest.java:
```diff
@@ -0,0 +1,45 @@
+    }
+
+    @Test
+    public void checkBurgerTabs() {
```
Здесь в одном методе написано три разных проверки. Надо их разделить по отдельным методам. Один тест должен проверять что-то одно. Если одна проверка упадёт, то другие даже не будут выполнены.

mkouznetsov-git commented on src/test/java/LoginTest.java:
```diff
@@ -0,0 +1,170 @@
+import org.junit.Before;
+import org.junit.Test;
+import org.openqa.selenium.WebDriver;
+import org.openqa.selenium.chrome.ChromeDriver;
```
Импорт не используется

mkouznetsov-git commented on src/test/java/MyAccountTest.java:
```diff
@@ -0,0 +1,89 @@
+import org.junit.Before;
+import org.junit.Test;
+import org.openqa.selenium.WebDriver;
+import org.openqa.selenium.chrome.ChromeDriver;
```
Импорт не используется

mkouznetsov-git commented on src/test/java/MyAccountTest.java:
```diff
@@ -0,0 +1,89 @@
+import stellarburgers.pages.AccountPage;
+import stellarburgers.pages.AuthForm;
+import stellarburgers.pages.ConstructorPage;
+import stellarburgers.pages.RegistrationForm;
```
Импорт не используется

mkouznetsov-git commented on src/test/java/RegisterTest.java:
```diff
@@ -0,0 +1,95 @@
+import org.junit.Before;
+import org.junit.Test;
+import org.openqa.selenium.WebDriver;
+import org.openqa.selenium.chrome.ChromeDriver;
```
Импорт не используется

Commit:
Исправила замечания:
-убрала неиспользуемые импорты
-разбила тест табов на три теста
-обновила allure отчет
